src/lib.rs

```diff
@@ -1,75 +1,22 @@
 use crate::common::*;
 
-use clap::Arg;
 use std::error::Error;
-use std::ffi::OsStr;
 use std::path::PathBuf;
-use std::{env, process};
 
 mod checks;
 mod common;
 mod fs_utils;
 
-fn get_args(current_dir: &OsStr) -> clap::ArgMatches {
-    clap::App::new(env!("CARGO_PKG_NAME"))
-        .about(env!("CARGO_PKG_DESCRIPTION"))
-        .author(env!("CARGO_PKG_AUTHORS"))
-        .version(env!("CARGO_PKG_VERSION"))
-        .version_short("v")
-        .arg(
-            Arg::with_name("input")
-                .help("files or paths")
-                .index(1)
-                .default_value_os(current_dir)
-                .required(true)
-                .multiple(true),
-        )
-        .arg(
-            Arg::with_name("exclude")
-                .short("e")
-                .long("exclude")
-                .value_name("FILE_NAME")
-                .help("Excludes files from check")
-                .multiple(true)
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("skip")
-                .short("s")
-                .long("skip")
-                .value_name("CHECK_NAME")
-                .help("Skips checks")
-                .multiple(true)
-                .takes_value(true),
-        )
-        .arg(
-            Arg::with_name("show-checks")
-                .long("show-checks")
-                .help("Shows list of available checks"),
-        )
-        .get_matches()
+pub fn show_checks() -> Vec<String> {
```

Comment:
I suggest use: 
```suggestion
pub use checks::available_check_names;
```
Because the name of the function(`show_checks`) indicates the action, but in fact it’s just a delegate.

Or make this function with `println`, but better stay all prints in `main.rs`

Reply:
Good catch 🙂